apps/nextjs-website/src/components/organisms/PostIntegration/PostIntegration.tsx

```diff
@@ -33,17 +34,34 @@ const PostIntegration = ({
   listTitle,
   serviceModels,
   guides,
+  backgroundVariant = 'white',
 }: PostIntegrationProps) => {
   const theme = useTheme();
   const t = useTranslations();
 
+  const backgroundStyles = {
+    white: {
+      backgroundColor: theme.palette.background.paper,
+      paddingBottom: 0,
+    },
+    lightGrey: {
+      backgroundColor: theme.palette.grey[50],
+      paddingBottom: 8,
+    },
+  };
```

Comment:
I'd memoize this as well 
```suggestion
const backgroundStyles = useMemo(() => ({
  white: {
    backgroundColor: theme.palette.background.paper,
    paddingBottom: 0,
  },
  lightGrey: {
    backgroundColor: theme.palette.grey[50],
    paddingBottom: 8,
  },
}), [theme]);
```